cmd/opentelemetry/app/receiver/jaegerreceiver/jaeger_receiver.go

```diff
@@ -87,15 +89,22 @@ func configureCollector(v *viper.Viper, cfg *jaegerreceiver.Config) {
 				Endpoint: cOpts.CollectorGRPCHostPort,
 			},
 		}
-		if cOpts.TLS.CertPath != "" && cOpts.TLS.KeyPath != "" {
-			cfg.GRPC.TLSSetting = &configtls.TLSServerSetting{
-				ClientCAFile: cOpts.TLS.ClientCAPath,
-				TLSSetting: configtls.TLSSetting{
-					KeyFile:  cOpts.TLS.KeyPath,
-					CertFile: cOpts.TLS.CertPath,
+	}
+	if cOpts.TLS.Enabled == true {
```

Comment:
can simplify `if cOpts.TLS.Enabled {`

Should enabling TLS through the Jaeger Collector options also turn on TLS for HTTP below?